dev/adb.ts

```diff
@@ -0,0 +1,780 @@
+const installOptions = [
```

Comment:
Hey alex, any chance you could type this so we can catch errors in the future? 
```suggestion
const installOptions:Fig.Option[] = [
```